null/primitive_test.go

```diff
@@ -262,7 +262,7 @@ func TestPrimitives(t *testing.T) {
 		}
 		if b, err := val.Value(); err != nil {
 			t.Error(err)
-		} else if b.(uint16) != 5 {
```

Comment:
What's the reason behind this change?

Reply:
In the documentation for `driver.Value`, it says that it is either nil, a type handled by the driver, or one of these types

- int64
- float64
- bool
- []byte
- string
- time.Time

While most drivers handle other integer types just fine, I think it is safest to just send `int64`. Same thing with `float64` for all floats.

Reply:
Did you get a chance to take another look?